Updates on README and project name (#15)
robotology/walking-teleoperation: docs/Dependencies.md

2. **YARP:** Using superbuild install the [YARP](http://www.yarp.it/): to handle the comunication with the robot with both ovrheadset and SDLjoypad drivers;

   + if you are not using superbuild, look at [this link](http://www.yarp.it/) for installing YARP.

>    + Use YARP and icub repos using the devel branch instead of the master branch. The main reason is beacuase of the `camCalibWithPose` application.

Reviewer: ```suggestion
   + Use YARP and icub repos using the devel branch instead of the master branch. The main reason is because of the `camCalibWithPose` application.
```

   + Enable the following options (be sure that this option is enabled in all the machines for the image communication):
   ```
   ENABLE_yarpcar_mjpeg
   MJPEG_AUTOCOMPRESS
   ```

3. **QT5 and Eigen3:** In the Super-build, enable the QT5 and Eigen3 to install (follow the instructions [here](https://github.com/robotology/robotology-superbuild#system-libraries)).

> 4. **Oculus SDK:** In order to install the oculus SDK, you need to install first the Oculus app on windows (oculus application), install the depenecies (GLEW, GLFW3), and finally install the sdk (LibOVR):

Reviewer: ```suggestion
4. **Oculus SDK:** In order to install the oculus SDK, you need to install first the Oculus app on windows (oculus application), install the dependencies (GLEW, GLFW3), and finally install the sdk (LibOVR):
```

   1. `oculus application`: Download and install the oculus [setup application](https://www.oculus.com/setup/). It is used for both the Oculus Virtual Reality headset (ovrheadset) and joypads (touch controller).
          

>    2. `GLEW`: Dowload the version 2.1.0 of the [glew libarary](http://glew.sourceforge.net/index.html) from this [link](https://sourceforge.net/projects/glew/files/glew/2.1.0/) (if you want to dowload from the [glew libarary](http://glew.sourceforge.net/index.html) select the source file)

Reviewer: ```suggestion
   2. `GLEW`: Download the version 2.1.0 of the [glew library](http://glew.sourceforge.net/index.html) from this [link](https://sourceforge.net/projects/glew/files/glew/2.1.0/) (if you want to download from the [glew library](http://glew.sourceforge.net/index.html) select the source file)
```
